packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/features/listFeatures.ts

```diff
@@ -248,15 +248,19 @@ function prepareAutoBullet(editor: IEditor, range: Range) {
     }
 }
 
-function toggleListAndPreventDefault(event: PluginKeyboardEvent, editor: IEditor) {
+function toggleListAndPreventDefault(
+    event: PluginKeyboardEvent,
+    editor: IEditor,
+    includeSiblingList: boolean = true
+) {
     let listInfo = cacheGetListElement(event, editor);
     if (listInfo) {
         let listElement = listInfo[0];
         let tag = getTagOfNode(listElement);
         if (tag == 'UL') {
             toggleBullet(editor);
         } else if (tag == 'OL') {
-            toggleNumbering(editor);
+            toggleNumbering(editor, null, includeSiblingList);
```

Comment:
I'm thinking about, if we should export that much parameters to a public format API (toggleNumbering). How about call toggleListType directly instead? So that no need to change the parameters for toggleNumbering.

And actually this bug happens for bullet list as well, it is just not that easy to repro. So maybe we should fix both of them, and calling toggleListType seems better here